user_manual/groupware/mail.rst

```diff
@@ -65,6 +65,23 @@ Mail can move a message to a different folder when it is marked as junk.
 
    .. figure:: images/mail_move-message-to-junk-folder.png
 
+Enable mail body search
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+   .. versionadded:: 3.5
+
+Mail bodies can now be searched, this feature is opt-in because of potential performance issues.
+
+To enable it: 
+
+1) Visit Account settings
+2) Go to Mailbox search
+3) Enable mail body search
+
+.. warning:: If you want to also enable it for unified mailboxes you have to do so in Mail settings
+
+By enabling it the main search box will now search in both subjects and mail bodies, and a sperate *Body* option 
+will appear in advanced search.
+
```

Comment:
"Once activated, the main search box will include searches in both subjects and email bodies. Additionally, a distinct body option will be available in advanced search."